### ISSUE TITLE
a11y: verificare link con disabilità visive in termini di colore

### ISSUE THREAD

**Issue by francescozaia** [italia/bootstrap-italia#323]
## Comportamento atteso
Con riferimento a https://www.w3.org/WAI/WCAG21/Techniques/failures/F73, dovremmo sempre usare delle differenziazioni di forma (e non soltanto al mouse over) per ogni link presente in pagina e nel kit.

## Comportamento attuale
Come ben sappiamo, l'utilizzo del solo colore non è sufficiente ad evidenziare i link di tipo `<a>`. C'è da verificare la presenza di questo problema nella libreria e poi, in cascata, anche su altri progetti (da discutere in seguito, lo scrivo qui come reminder).
